### PR TITLE
Skip unneeeded code block

### PR DESCRIPTION
**Pull request by SCdF**
Skip executing a code block where there is nothing to do

#6223

### CODE REVIEW

**Comment by ngaruko** [comment]
We need a reviewer on this @SCdF ...

**Comment by SCdF** [comment]
@ngaruko oh lol interesting, Willy reviewed it but because he's no longer in the org it's no longer valid? Feel free to continue with AT, I'll get another reviewer on it.

File: sentinel/src/schedule/outbound.js [hunk 1]
```diff
@@ -271,21 +271,25 @@ const singlePush = (taskDoc, medicDoc, config, key) => Promise.resolve()
   });
 
 const removeInvalidTasks = invalidTasks => {
-  logger.warn(`Found ${invalidTasks.length} tasks that could not have their associated records loaded:`);
+  if (invalidTasks.length) {
```
Comment (dianabarsan) [inline sentinel/src/schedule/outbound.js:274]: Please consider returning early:

```
if (!invalidTasks.length) {
    return Promise.resolve();
}
```

**Comment by SCdF** [comment]
I have re-implemented this change as part of the refactoring done in #6306

**Pull request closed without merging**